CredentialProvider.Microsoft/Util/SessionTokenCache.cs

```diff
@@ -4,15 +4,16 @@
 
 using System;
 using System.Collections.Generic;
+using System.IO;
 using Newtonsoft.Json;
+using NuGetCredentialProvider.Logging;
 
 namespace NuGetCredentialProvider.Util
 {
     public class SessionTokenCache : ICache<Uri, string>
```

Comment:
Can we add logger to this class as a variable and not in the interface method? TryGetValue(Uri key, out string value) looks more correct than having to pass a logger.

Reply:
Good idea - done!